src/ui/components/NetworkMonitor/RequestTable.module.css

```diff
@@ -62,7 +63,8 @@ table.requests {
   border-bottom-right-radius: 8px;
   border-top-right-radius: 8px;
   cursor: pointer;
-  display: none;
```

Comment:
With display none the fast-forward/rewind buttons is not accesible, I swap this, to use a opacity